lib/volapi.js

```diff
@@ -0,0 +1,150 @@
+var util = require('util');
```

Comment:
license and copyright block missing?

Please have two blocks of requires: external packages and then local libs.